arch/cpu/cc26xx-cc13xx/rf-core/prop-mode.c

```diff
@@ -795,6 +796,12 @@ read_frame(void *buf, unsigned short buf_len)
     entry->status = DATA_ENTRY_STATUS_PENDING;
   }
 
+  if(rx_is_full) {
+    rx_is_full = false;
+    printf("RXQ was full, re-enabling radio!\n");
+    rx_on_prop();
```

Comment:
We may have to think of a better logic here in the future, but since we are only using CSMA this will do for now I think